navigation/TabNavigation.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { Ionicons } from '@expo/vector-icons';
+
+import BasalTestingScreen from '../screens/BasalTestingScreen';
+import DashboardScreen from '../screens/DashboardScreen';
+import AddFoodScreen from '../screens/AddFoodScreen';
+import MonthLogScreen from '../screens/MonthLogScreen';
+
+const Tab = createBottomTabNavigator();
+
+const BottomTabNavigator = () => {
+  return (
+    <Tab.Navigator screenOptions={{ headerShown: false }}>
+      <Tab.Screen name="Home" component={DashboardScreen} options={{ tabBarIcon: ({color,size}) => (<Ionicons name='home' color={color} size={size}/>) }}/>
+      <Tab.Screen name="Basal" component={AddFoodScreen} options={{ tabBarIcon: ({color,size}) => (<Ionicons name='nutrition' color={color} size={size}/>) }}/>
+      <Tab.Screen name="History" component={MonthLogScreen} options={{ tabBarIcon: ({color,size}) => (<Ionicons name='calendar' color={color} size={size}/>) }}/>
+      <Tab.Screen name="Testing" component={BasalTestingScreen} options={{ tabBarIcon: ({color,size}) => (<Ionicons name='heart-circle' color={color} size={size}/>) }} />
+    </Tab.Navigator>
+  )
```

Comment:
Overall, I like how the navigation is being set up in this project. Just make some changes to use Context or redux rather than setting specific variables to global variables, also look into using Navigate and protecting your routes based on variables that can be passed as props. Take a reference at Sarah's authentication and authorization tutorial to better understand how to guard your routes and use Navigate. Great job, this project has a lot of potential. I look forward to downloading and using it on my phone.